autogluon/utils/tabular/contrib/tabular_nn_pytorch/hyperparameters/parameters.py

```diff
@@ -0,0 +1,43 @@
+from autogluon.utils.tabular.ml.constants import BINARY, MULTICLASS, REGRESSION
+
+
+def get_param_baseline(problem_type, num_classes=None):
+    if problem_type == BINARY:
+        return get_param_binary_baseline()
+    elif problem_type == MULTICLASS:
+        return get_param_multiclass_baseline()
+    elif problem_type == REGRESSION:
+        return get_param_regression_baseline()
+    else:
+        return get_param_binary_baseline()
+
+
+def get_param_multiclass_baseline():
+    params = {
```

Comment:
Can you add comment linking to fastai source code/docs for users to learn what these hyperparameters mean?

Reply:
Also why is: 'nn.tabular' appearing in every hyperparameter name? I presume these hyperparameters are all anyway specific to this model, so this makes their names overly confusing IMO.  I'd just use the exact same names as in the fastai library so users can just look at its documentation.

Reply:
- added docs/links
- removed 'nn.tabular.' prefix.